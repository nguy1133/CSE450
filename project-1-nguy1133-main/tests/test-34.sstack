// ERROR: Variable names may not begin with a number.
PUSH "7Seas"
POP 7seas
