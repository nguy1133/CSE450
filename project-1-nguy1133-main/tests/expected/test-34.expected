ERROR (line 3): Unexpected token '7seas'
